.github/workflows/pypi-publish.yml

```diff
@@ -21,6 +24,13 @@ jobs:
       run: >-
         python -m pip install -U twine wheel
 
+    - name: Change the version file for TestPyPI upload
+      if: (github.repository == 'optuna/optuna') && (github.event_name == 'schedule')
+      run: |
+        OPTUNA_VERSION=$( cat optuna/version.py )
+        DATE=`date +"%Y%m%d"`
+        echo __version__\ =\ \"${OPTUNA_VERSION}${DATE}\" > optuna/version.py
```

Comment:
In my environment, `optuna/version.py` was updated as follows:
```python
__version__ = "__version__ = "2.5.0.dev"20210121"
```

I guess we need to extract only the version string from `optuna/version.py`. In addition, we can reduce the escape characters in L32.

```suggestion
        OPTUNA_VERSION=$(cut -d '"' -f 2 optuna/version.py)
        DATE=`date +"%Y%m%d"`
        echo "__version__ = \"${OPTUNA_VERSION}${DATE}\"" > optuna/version.py
```

Reply:
Ops! I had made the wrong fix when it was changed to the dev version. Thank you. PTAL.